### PR TITLE
Add note regarding expiration of Tuya IoT Platform trial period.

### PR DESCRIPTION
**Pull request by creeve4**

### CODE REVIEW

File: source/_integrations/tuya.markdown [hunk 1]
```diff
@@ -57,6 +57,7 @@ All Home Assistant platforms are supported by the Tuya integration, except the l
 - Your devices need first to be added in the [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
 - You will also need to create an account in the [Tuya IoT Platform](https://iot.tuya.com/).
 This is a separate account from the one you made for the app. You cannot log in with your app's credentials.
+**Note**: this account may have a trial period, which expires by default after 6 months and must be manually renewed. Once expired, all devices will be revoked and unavailable in Home Assistant. Renewal may take several days.
```
Comment (frenck) [inline source/_integrations/tuya.markdown:60]: That isn't a prerequisite, this fits more into the troubleshooting section below.

**Review by frenck — CHANGES_REQUESTED** [review]
I've noticed you've removed the PR template in this PR. Please restore it and fill it out.

Thanks! 👍 

../Frenck

**Comment by creeve4** [comment]
> I've noticed you've removed the PR template in this PR. Please restore it and fill it out.
> 
> Thanks! 👍
> 
> ../Frenck

My mistake. Where can I find the template (without creating a new PR)?

**Comment by frenck** [comment]
> Where can I find the template (without creating a new PR)?

Just open up a new PR (without creating a new PR, just get into the dialog), and copy paste the template shown.

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

**Pull request closed without merging**